code/game/objects/items/devices/window_painter.dm

```diff
@@ -0,0 +1,22 @@
+/obj/item/pipe_painter/window_painter
+	name = "window painter"
+	icon_state = "window_painter"
+
+var/list/paintable_windows = list(
+		/obj/structure/window/reinforced,
+		/obj/structure/window/basic,
+		/obj/structure/window/full/reinforced,
+		/obj/structure/window/full/basic,
+		/obj/machinery/door/window)
```

Comment:
```suggestion
	var/list/paintable_windows = list(
			/obj/structure/window/reinforced,
			/obj/structure/window/basic,
			/obj/structure/window/full/reinforced,
			/obj/structure/window/full/basic,
			/obj/machinery/door/window)
```
Needs to be indented to be part of the definition, not a global (unindented) list